### DISCUSSION TITLE
I can't join in the pool

### DISCUSSION THREAD

**Discussion by SirHunterBG** [Chia-Network/chia-blockchain#8634] [Support]
There is a question. I try to join the pool, but I don't get the minimum chia coin. How long do I have to wait to get the necessary chia coin to be able to join the pool?

**Comment by SirHunterBG** [comment]
![Uploading 1.jpg…]()

**Comment by SirHunterBG** [comment]
![Uploading 2.jpg…]()

**Comment by SirHunterBG** [comment]
![Uploading 3.jpg…]()

**Comment by loppefaaret** [comment]
there is several chia faucets out there, that you can use to get initial coins to use

**Comment by BrandtH22** [comment]
@SirHunterBG , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
